src/foam/nanos/servlet/VirtualHostRoutingServlet.js

```diff
@@ -42,34 +74,173 @@ foam.CLASS({
+          hasClosingTag = (Boolean) tagConfig.get("hasClosingTag");
+        }
+
+        Boolean isSingleClosingTag = false;
+        if ( tagConfig.containsKey("isSingleClosingTag") ) {
+          isSingleClosingTag = (Boolean) tagConfig.get("isSingleClosingTag");
+        }
+
+        String headTag = this.getTagString((String) tagConfig.get("tag"), keywords, (HashMap) tagConfig.get("attributes"), hasClosingTag, isSingleClosingTag);
+        out.println(headTag);
+      }
+      
+      if ( this.getIsResourceStorage() ) {
+        out.println("<script language=\\"javascript\\" src=\\"/foam-bin-@VERSION@.js\\"></script>");
+        out.println("<script async defer language=\\"javascript\\" src=\\"/html2canvas.min.js\\"></script>");
+        out.println("<script async defer language=\\"javascript\\" src=\\"/jspdf.min.js\\"></script>");
+        out.println("<script language=\\"javascript\\" src=\\"/jspdf.plugin.autotable.min.js\\"></script>");
+        out.println("<script async defer language=\\"JavaScript\\" src=\\"https://cdn.plaid.com/link/v2/stable/link-initialize.js\\"></script>");
+      }
+      else {
+        if ( this.getLiveScriptBundlerDisabled() ) {
+          out.println("<script language=\\"javascript\\" src=\\"../../../../foam2/src/foam.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../foam2/src/foam/nanos/nanos.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../foam2/src/foam/support/support.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../nanopay/src/net/nanopay/files.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../nanopay/src/net/nanopay/iso20022/files.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../nanopay/src/net/nanopay/iso8583/files.js\\"></script>");
+          out.println("<script language=\\"javascript\\" src=\\"../../../../nanopay/src/net/nanopay/flinks/utils/files.js\\"></script>");
+        }
+        else {
+          out.println("<script language=\\"javascript\\" src=\\"/service/liveScriptBundler\\"></script>");
+        }
+        out.println("<script async defer language=\\"JavaScript\\" src=\\"https://cdn.plaid.com/link/v2/stable/link-initialize.js\\"></script>");
+        out.println("<script async defer language=\\"javascript\\" src=\\"../../../../node_modules/html2canvas/dist/html2canvas.min.js\\"></script>");
+        out.println("<script language=\\"javascript\\" src=\\"../../../../node_modules/jspdf/dist/jspdf.min.js\\"></script>");
+        out.println("<script async defer language=\\"javascript\\" src=\\"../../../../node_modules/jspdf-autotable/dist/jspdf.plugin.autotable.min.js\\"></script>");
+      }
+      out.println("<link href=\\"https://fonts.googleapis.com/css?family=Roboto:100,300,400,500\\" rel=\\"stylesheet\\">");
```

Comment:
are we still able to override/replace these?

Reply:
@jlhughes No, not as it is right now. Alternatively, we could off-load the font specifications to Theme in the same way we did favicons.